src/main/scala/cromwell/binding/RuntimeAttributes.scala

```diff
@@ -0,0 +1,86 @@
+package cromwell.binding
+
+import com.google.api.services.genomics.model.Disk
+import cromwell.binding.AstTools.{AstNodeName, EnhancedAstNode}
+import cromwell.binding.RuntimeAttributes.{Defaults, LocalDisk}
+import cromwell.parser.MemorySize
+import cromwell.parser.WdlParser.{Ast, AstList}
+
+import scala.collection.JavaConverters._
+import scala.language.postfixOps
+
+object RuntimeAttributes {
+  /** Convenience class to convert to Google `Disk` class. */
+  private case class LocalDisk(name: String, sizeGb: Long, diskType: String) {
+    def toDisk: Disk = new Disk().setSizeGb(sizeGb).setType(diskType).setName(name)
+  }
+
+  /** Fallback values if values for these keys are not specified in a task "runtime" stanza. */
+  object Defaults {
+    val Cpu = 1
+    val Disk = Seq(LocalDisk("local-disk", 100L, "LOCAL_SSD").toDisk)
+    val FailOnStderr = false
+    val MemoryInBytes = MemorySize.GB.toBytes(2)
+    val Preemptible = false
+    val Zones = Array("us-central1-a")
+  }
+
+  def apply(ast: Ast): RuntimeAttributes = {
+    def processRuntimeAttribute(ast: Ast): (String, String) = {
+      (ast.getAttribute("key").sourceString(), ast.getAttribute("value").sourceString())
+    }
+
+    def processRuntimeAttributes(astList: AstList): Map[String, String] = {
+      astList.asScala.toVector map { a => processRuntimeAttribute(a.asInstanceOf[Ast]) } toMap
+    }
+
+    val asts = ast.findAsts(AstNodeName.Runtime)
+    if (asts.size > 1) throw new UnsupportedOperationException("Only one runtime block may be defined per task")
+    val astList = asts.headOption map {
+      _.getAttribute("map").asInstanceOf[AstList]
+    }
+    val attributeMap = astList map processRuntimeAttributes getOrElse Map.empty[String, String]
+
+    RuntimeAttributes(attributeMap)
+  }
+}
+
+/**
+ * Parses configuration values per WDL/JES specs.  Constructor is private to force use of companion object
+ * `apply` method.
+ */
+case class RuntimeAttributes private(attributes: Map[String, String]) {
+
+  def attribute(attr: String): Option[String] = attributes.get(attr)
```

Comment:
If it's private then I really don't see much point to having it. I thought you intended this to be a convenience to outside clients of the class and then question is if `foo.attribute("blah")` read better than `foo.get("blah")`